src/Pmi/WorkQueue/WorkQueue.php

```diff
@@ -9,6 +9,7 @@ class WorkQueue
     const HTML_SUCCESS = '<i class="fa fa-check text-success" aria-hidden="true"></i>';
     const HTML_DANGER = '<i class="fa fa-times text-danger" aria-hidden="true"></i>';
     const HTML_WARNING = '<i class="fa fa-question text-warning" aria-hidden="true"></i>';
+    const HTML_NOTICE = '<i class="fa fa-exclamation-circle text-warning" aria-hidden="true"></i>';
```

Comment:
The ticket says to use an "open circle". I suggest that we use `fa-stop-circle` (I think the `fa-circle-o` icon is too thin).